account/applications/methods/patchApplicationsApplicationId.md

```diff
@@ -13,14 +13,22 @@
-| `CallStatusCallbackUrl`    | No        | Url to receive [voice events](../../../voice/bxml/callbacks/about.md) **NOT** related to Initiated. Such as: rejected or hung up.                                              |
-| `CallStatusMethod`         | No        | HTTP method for events sent to the `CallStatusCallbackUrl`.<br> <code class="post">POST</code> or <code class="get">GET</code><br>Default is <code class="post">POST</code>    |
+| Parameters                            | Mandatory | Description                                                                                                                                                                    |
+|:--------------------------------------|:----------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `AppName`                             | No        | Plain text name of the application                                                                                                                                             |
+| `MsgCallbackUrl`                      | No        | Url to receive [message events](../../../messaging/callbacks/messageEvents.md)                                                                                                 |
+| `CallInitiatedCallbackUrl`            | No        | Url to receive [voice events](../../../voice/bxml/callbacks/about.md)                                                                                                          |
+| `CallInitiatedMethod`                 | No        | HTTP method for events sent to the `CallInitiatedCallbackUrl`.<br> <code class="post">POST</code> or <code class="get">GET</code><br>Default is <code class="post">POST</code> |
+| `CallStatusCallbackUrl`               | No        | Url to receive [voice events](../../../voice/bxml/callbacks/about.md) **NOT** related to Initiated. Such as: rejected or hung up.                                              |
+| `CallStatusMethod`                    | No        | HTTP method for events sent to the `CallStatusCallbackUrl`.<br> <code class="post">POST</code> or <code class="get">GET</code><br>Default is <code class="post">POST</code>    |
+| `CallbackCreds`                       | No        | Basic auth credentials to apply to your message & voice events                                                                                                                 |
+| `CallbackCreds.UserId`                | No        | Basic auth `UserId`                                                                                                                                                            |
+| `CallbackCreds.Password`              | No        | Basic auth `Password`                                                                                                                                                          |
```

Comment:
Interesting that these were missing. The rules for which fields are required under which circumstances are a little more complicated than the description indicates, (I _think_ `Password` is required if you're trying to change the `UserId`). Not sure if we need to spell out those combinations, or let the error-message guide the user's experience. 🤔 

Maybe it's as simple as changing the `No` on `Password` to `No, required if CallbackCreds.UserId is supplied`

Thoughts?